### PR TITLE
Move bkg renamer to function

### PR DESCRIPTION
**Pull request by AndrewEichmann-NOAA**
Moves bit of code to rename background files so that it can be used for centering task

Works towards addressing https://github.com/NOAA-EMC/GDASApp/issues/912

### CODE REVIEW

File: ush/soca/bkg_utils.py [hunk 1]
```diff
@@ -157,3 +158,19 @@ def gen_bkg_list(bkg_path, out_path, window_begin=' ', yaml_name='bkg.yaml', ice
...
     # copy ocean backgrounds to RUNDIR
     FileHandler({'copy': bkg_list_src_dst}).sync()
+
+
+def rename_bkg(bkg_dir, anl_dir, RUN):
+
+    # Copy and rename initial condition
+    ics_list = []
+    # ocean IC's
+    mom_ic_src = glob.glob(os.path.join(bkg_dir, f'{RUN}.ocean.*.inst.f003.nc'))[0]
```
Comment (CoryMartin-NOAA) [inline ush/soca/bkg_utils.py]: why are we globbing here?

Comment (AndrewEichmann-NOAA) [reply ush/soca/bkg_utils.py]: @guillaumevernieres ?

Comment (CoryMartin-NOAA) [reply ush/soca/bkg_utils.py]: Does this file name ever vary? and if so, is it predictable? like is it date/time based? If this is a crutch to not use 't06z' then we need to include that here.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Pull request merged**